Automattic/wp-calypso
# Like button: clean up component

This PR cleans up the like button component and renames the misleading `showCount` prop to `showZeroCount`.

Todo
- [x] ES6-ify components
- [x] Rename confusing prop `showCount` https://github.com/Automattic/wp-calypso/issues/6767

Fixes #6767.

Test live: https://calypso.live/?branch=fix/reader/like-button-spring-clean

### To test

Ensure that the like button displays correctly in posts, search and comment likes.
